### PR TITLE
Fix for confetti

### PR DESCRIPTION
**Pull request by icexuick**
I hope!

All colormap codes should be 4 numbers now.
Haven't tested in-game yet.

### CODE REVIEW

**Comment by Ruwetuin** [comment]
now the 2nd and 3rd color have 5 instead of 4 values (the 3rd one of each of these (0.01) must be removed)

File: effects/genericunitexplosion.lua [hunk 1]
```diff
@@ -3058,7 +3058,7 @@ definitions['decoycommander'].confetti = {
 	unit               = false,
 	properties = {
 		airdrag            = 0.95,
-		colormap           = [[1 0 0 0.01  0 1 0 0.01 0.1  1 0 0 0.01 0.1  0 1 0.5 0.01  1 0 0.01  0 1 0 0.01  1 1 0 0.01  1 0 0.01   1 0.6 0.01  0 1 0 0.01 0.1   0.1 0.7 0.01   0.6 0.3 0.6 0.01   0.6 0.6 0.6 0.01   0 0 0 0.01  ]],
+		colormap           = [[1 0 0 0.01  0 1 0 0.01 0.1  1 0 0 0.01 0.1  0 1 0.5 0 0.01  1 0 0 0.01  0 1 0 0.01  1 1 0 0.01  1 0 0 0.01   1 0.6 0 0.01  0 1 0 0.01 0.1   0.1 0.7 0 0.01   0.6 0.3 0.6 0.01   0.6 0.6 0.6 0.01   0 0 0 0.01  ]],
```
Comment (sprunk) [inline effects/genericunitexplosion.lua:3061]: you've got a bunch of groups of 5 and are introducing a new one
image

**Comment by Beherith** [comment]
Dont merge, superceded by https://github.com/beyond-all-reason/Beyond-All-Reason/pull/3651

File: effects/genericunitexplosion.lua (hunk 1 above)
Comment (icexuick) [reply effects/genericunitexplosion.lua:3061]: Thank you sprunk! It was the end of the day!
Shit, now my hands are dirty!

**Pull request closed without merging**